### ISSUE TITLE
Prettier and Lint

### ISSUE THREAD

**Issue by GuillouTran** [telmogoncalves/telmo#15]
Dear @telmogoncalves,

As the fact that we should all shipping quality codes, would you consider integrating eslint and prettier?

**Comment by telmogoncalves** [comment]
Yeah, that's something I usually use on most of my projects. I'll try to take care of it when I have time.

**Comment by GuillouTran** [comment]
May you have a look and give me some suggestions and hint to setup on my project? I been ransacking the web but I am not so sure on what kind of prettier or syntax you’re using! Plus the fact that I am brand new to NextJS and they obviously don’t offered to much documentation on testing like Gatsby and such. So, may you please spend sometimes and give me some suggestion? It would be more than appreciates if you make a pull request!

**Comment by telmogoncalves** [comment]
I'm not gonna make a pull request, that goes against all principles of your learning.

Here are some resources:

- [https://www.npmjs.com/package/eslint-config-nextjs](https://www.npmjs.com/package/eslint-config-nextjs)
- [https://stackoverflow.com/questions/58233482/next-js-setting-up-eslint-for-nextjs](https://stackoverflow.com/questions/58233482/next-js-setting-up-eslint-for-nextjs)

**Comment by GuillouTran** [comment]
I understand! Then, may I make a pull request then you have a look on it see where I went wrong and give me some feedback?

**Comment by telmogoncalves** [comment]
Yeah, that would do it. Let me know when you have it ready for review.

**Comment by GuillouTran** [comment]
Great! Working on it!

**Comment by GuillouTran** [comment]
Hi @telmogoncalves,
This is my first attempt, kinda terrible. Have a look, would you?
https://github.com/GuillouTran/guilloutran.com/pull/63
